src/Employer/Employer.Web/Controllers/VacancyManageController.cs

```diff
@@ -32,6 +33,25 @@ public VacancyManageController(VacancyManageOrchestrator orchestrator)
             return View(m.ViewName, m.ViewModel);
         }
 
+        [HttpGet("view", Name = RouteNames.DisplayFullVacancy_Get)]
+        public async Task<IActionResult> DisplayFullVacancy(VacancyRouteModel vrm)
+        {
+            var vacancy = await _orchestrator.GetVacancy(vrm);
+
+            if (vacancy.CanEdit)
+            {
+                return HandleRedirectOfDraftVacancy(vacancy);
+            }
+
+            if (!string.IsNullOrEmpty(vacancy.ApplicationUrl) || (vacancy.Status != VacancyStatus.Live && vacancy.Status != VacancyStatus.Closed))
```

Comment:
Currently the only reason for that page is for vacancies that have applications. It is odd but it is what  was asked. It would be simpler if the requirement was different. It is tricky because we need to have decisions around which preview to show when a vacancy is clicked on from the dashboard.